examples/dev/Dashboard.html

```diff
@@ -29,21 +27,12 @@
         text-align: right;
         border: 2px solid purple;
       }
-
-      ul {
-        margin: 60px;
-      }
-
-      li {
-        margin: 60px;
-      }
-
-      .foo a {
-        color: purple;
-      }
```

Comment:
These were added to test Uppy’s style protection from some styles that might be on a page where Uppy is used. Not sure it’s the best idea, but just noting that there was a reason, maybe we should keep it.

Reply:
We absolutely should! I'll readd it and add the comment.